src/main/java/model/janggi_board/JanggiBoard.java

```diff
@@ -0,0 +1,110 @@
+package model.janggi_board;
+
+import static model.janggi_board.JanggiBoardSetUp.DEFAULT_SETUP;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import model.Path;
+import model.Point;
+import model.Team;
+import model.piece.Piece;
+
+public class JanggiBoard {
+    public static final int VERTICAL_SIZE = 10;
+    public static final int HORIZONTAL_SIZE = 9;
+    private ArrayList<ArrayList<Dot>> janggiBoard;
```

Comment:
자료구조를 ArrayList 로 제한할 필요가 없어 보이네요~

```suggestion
    private List<List<Dot>> janggiBoard;
```

Reply:
이런 부분이 SOLID 원칙을 위반하는 부분일 수 도 있겠군요!